test/integration_test/s3/getObject.spec.js

```diff
@@ -0,0 +1,209 @@
+var helpers = require('../helpers');
+var AWS = helpers.AWS;
+var bucketName = helpers.sharedBucket;
+describe('download integrity', function() {
+  var params;
+  var s3;
+  function putObject(customParams, done) {
+    params = {
+      Bucket: customParams.Bucket || params.Bucket,
+      Key: customParams.Key || params.Key,
+    };
+    s3.createBucket({Bucket: params.Bucket}).promise().then(function() {
+      return s3.waitFor('bucketExists', {Bucket: params.Bucket}).promise()
+    }).then(function() {
+      return s3.putObject({
+        Bucket: params.Bucket,
+        Key: params.Key,
+        Body: customParams.Body || params.Body
+      }).promise();
+    }).then(function() {
+      return s3.waitFor('objectExists', {
+        Bucket: params.Bucket,
+        Key: params.Key,
+      }).promise();
+    }).then(function() { 
+      if (typeof done === 'function') done();
+    }).catch(function(err) {
+      throw new Error('Cannot put object: ' + err);
+      exit(1);
```

Comment:
Is `exit` exposed as a global? I thought it was `process.exit()`, and am not seeing `exit` in the node repl.

Also think this line is skipped since you're throwing an error right before it.